### DISCUSSION TITLE
CS-HOME 커뮤니티를 오픈하였습니다.

### DISCUSSION THREAD

**Discussion by gabrielyoon7** [KGU-CS-HOME/.github#2] [1. 공지사항]
# 환영합니다.

안녕하세요. CS-HOME에서 5~6기 팀장을 맡았던 17학번  윤주현(@gabrielyoon7) 입니다.

CS-HOME의 Discussions를 오픈 한 기념으로 여러 가지 소개를 하려고 합니다.

### 조직으로 이전 한 이유가 무엇인가요?
- 기존에는 프로젝트들이 개인 계정의 Repository로 존재하였으나, **2022년 부로 조직으로 관리**하게 되었습니다.
- 과거와 다르게 CS-HOME에서 여러 프로젝트를 관리해야 하기 때문이었습니다.
- 이 프로젝트 특성 상 언제 **인수 인계가 끊길 지 모르는 위험**이 있었습니다.
- 이 조직 페이지와 Discussion은 이제 여러분들이 외부로 홍보하거나 자료를 공유하기 위해 사용해주셨으면 합니다. 폐쇄적인 활동에서 조금 더 개방적이고 체계적이고 활발한 활동으로 이어졌으면 합니다.

### Issue기능도 있는데 Disccusions를 오픈 한 이유가 무엇인가요?
- 모든 활동을 기록으로 남기기 위해서는 하나의 저장소가 필요했습니다. 
- 모든 프로젝트가 Private로 관리되고 있어 Issue는 외부에 저희의 활동을 알리기가 어렵습니다.
- 하지만 Discussions는 전체 공개가 되어있으므로 **새로운 기수를 모집할 때 많은 사람들이 참고할 수 있을 것으로 기대**합니다.
- 더 나아가서 포트폴리오로 활용하고 싶은 사람들도 있을 것이라고 생각했습니다.
- 현재 존재하는 프로젝트들은 하나의 Core한 구조를 가지고 있어, 어느 한 쪽에서 발생하는 문제가 아닐 가능성이 있습니다. 따라서 Discussions에서 모아놓고 토의를 하는 것이 훨씬 나을 것이라고 판단했습니다.

### 어떻게 활용하면 될까요?
- **보안에 관련된 문제는 이 곳에서 토의하지 않았으면 합니다.** 위험한 내용은 업데이트가 된 이후, 이 곳에 공개했으면 합니다.
- 코드가 워낙 이 사람 저 사람 거쳐서 오다 보니 코드 리팩토링이 굉장히 시급합니다. 당장은 시간과 인력이 부족하여 진행하지 못할 것 같은 문제들을 기록해두시면 좋을 것 같습니다.
- 기록한 문제들을 어떤 방식으로 처리했는지 기록하시면서 **글 쓰는 능력**을 키우는 것은 물론이고, 후임자들이 학습하는데 많은 도움이 되길 바랍니다.

## 카테고리는 어떻게 쓰면 될까요?

> 사실 정해진 사용 법은 없습니다만, 다음과 같은 목적으로 카테고리를 생성했습니다.

1. 공지사항은 조직원이라면 누구나 작성할 수 있습니다. **공지할만한 내용을 공지로 작성**합니다. ex) 커뮤니티 관련 공지, 신입 기수 모집 등
2. 업데이트는 프로젝트의 변경 사항을 공지해주셨으면 합니다. 여기에 기록된 내용은 **후임자들 혹은 외부인이 참고할만한 굉장한 자료가 될 것**입니다.
3. 이슈는 프로젝트에 현재 존재하는 문제를 기록하는 곳 입니다. 단, **보안에 관련된 사항은 기록하지 않았으면 하고, 해결이 된 이후에 작성**해주셨으면 합니다.
4. 잡담은 그냥 아무 말이나 나눴으면 해서 만들었습니다.
5. 제안은 **이런 저런 아이디어를 기록**했으면 해서 만들었습니다. 학과 홈페이지에 이런 기능이 들어오면 좋겠다 싶은 것들을 기록하시거나 실제로 프로젝트로 이어가도 좋습니다. 실제로 과거에 사물함 신청 기능이나, 졸업요건진단 프로그램을 개발하여 업데이트하기도 했습니다.
6. Q&A는 외부인들이 이 프로젝트에 관심을 가질 수도 있어 열어놨습니다. **경기대학교 컴퓨터공학부 학생이라면 누구나 작성이 가능**합니다.

---

여러분의 활동이 하나의 유산으로 남기를 바라면서 커뮤니티를 오픈하였습니다.

**많.관.부**

**Comment by noahyunjun** [comment]
감사합니다
